test/conformance/exp_command_buffer/fixtures.h

```diff
@@ -112,24 +112,52 @@ struct urUpdatableCommandBufferExpExecutionTest
             GTEST_SKIP() << "Updating EXP command-buffers is not supported.";
         }
 
+        // Currently level zero driver doesn't support updating execution range.
+        // Also disable immediate command lists because there is synchronization issue causing test failures.
+        if (backend == UR_PLATFORM_BACKEND_LEVEL_ZERO) {
+            setenv("UR_L0_USE_IMMEDIATE_COMMANDLISTS", "0", 0);
```

Comment:
Do we need to set this environment variable if we're setting `UR_QUEUE_FLAG_SUBMISSION_BATCHED` later on when creating the queue?

Using env vars is a bit messy if users that setting their own too, so would like to avoid it in preference of setting properties on the queues if possible.

Reply:
Oh, sorry, I forgot to remove that, will do. You are right, it is not needed anymore.

Reply:
Removed.